src/index.js

```diff
@@ -1,24 +1,58 @@
-import './style.css';
-
-document.addEventListener('DOMContentLoaded', () => {
-  const scoreList = document.querySelector('.score-list');
-
-  const LeaderBoardData = [
-    { name: 'Manchester City', score: 12 },
-    { name: 'Tottenham Hotspur', score: 10 },
-    { name: 'Liverpool', score: 10 },
-    { name: 'Westhan United', score: 10 },
-  ];
-
-  const displayLeaderBoard = () => {
-    scoreList.innerHTML = '';
-
-    LeaderBoardData.forEach((team) => {
-      const listItem = document.createElement('li');
-      listItem.className = 'score-item';
-      listItem.textContent = `${team.name}: ${team.score}`;
-      scoreList.appendChild(listItem);
-    });
-  };
-  displayLeaderBoard();
-});
\ No newline at end of file
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/7UyWMdCgHZ87wkTWCz0G/scores/';
+
+async function fetchLeaderboard() {
+  try {
+    const response = await fetch(apiUrl);
+    const data = await response.json();
+
+    if (response.ok) {
+      const scoreList = document.getElementById('score-list');
+      scoreList.innerHTML = '';
+
+      data.result.forEach((score, index) => {
+        const listItem = document.createElement('li');
+        listItem.innerHTML = `${index + 1}. ${score.user} - ${score.score}`;
+        scoreList.appendChild(listItem);
+      });
+    } else {
+      Error('Failed to fetch leaderboard data');
+    }
+  } catch (error) {
+    Error('Error:', error);
+  }
+}
+
+async function handleSubmit(event) {
+  event.preventDefault();
+
+  const playerName = document.getElementById('player-name').value;
+  const playerScore = parseInt(document.getElementById('player-score').value, 10);
+
+  if (playerName && !Number.isNaN(playerScore)) {
+    try {
+      const response = await fetch(apiUrl, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ user: playerName, score: playerScore }),
+      });
+
+      if (response.ok) {
+        document.getElementById('player-name').value = '';
+        document.getElementById('player-score').value = '';
+
+        fetchLeaderboard();
+      } else {
+        Error('Failed to submit the score');
+      }
+    } catch (error) {
+      Error('Error:', error);
+    }
+  }
+}
+
+document.querySelector('.refresh-button').addEventListener('click', fetchLeaderboard);
+document.getElementById('score-form').addEventListener('submit', handleSubmit);
+
+fetchLeaderboard();
```

Comment:
- [ ] Great job so far @Obote However per the project requirements, you were asked to ensure the usage of **arrow functions** instead of the `function` keyword and this hasn't been implemented yet so kindly modify your code to meet the project requirements :wink: